index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <style>
+    @media (max-width: "700px"){
+  #projects {
+    width: 25%;
+  }
+}
+
+#welcome-section {
+  height: 100vh;
+}
+
+#navbar {
+  position: fixed;
+  top: 0;
+  right: 0;
+}
+</style>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section id="welcome-section">
+    <h1>These are some of my projects</h1>
+  </section>
+  <section id="projects">
+    <div class="project-tile">
+      <ul>
+        <li><a href="https://codepen.io/freeCodeCamp/full/zNqgVx"> </li>
+      </ul>
+    </div>
+    <nav id= "navbar">
+      <a href="#">
+      <a id="profile-link" href="#" target="_blank">
```

Comment:
You can use an unordered list for this links to group them.

Reply:
@Alibaba2023 👋 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use an unordered list for these links to group them.".
 
- [ ] Kindly use reasoning `why` students need to consider your suggestion. Why do students need to use an unordered list for these links to group them? Use reasoning here and you can add any referral link from where students can get hints.

- [ ] Kindly use [Grammarly](https://app.grammarly.com/) so that there is no spelling mistake and no capital letters in the middle of any sentence. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.